configs/rt_slam_plugins.yaml

```diff
@@ -4,10 +4,10 @@ plugin_group:
   - name: Kimera-VIO/
     path:
       git_repo: https://github.com/ILLIXR/Kimera-VIO
-      version: 6ea99d33fc2435a8c514049416dcb2f8a0e31692
+      version: "2.1"
```

Comment:
It hasn't but this is an update to be merged that exists in the Kimera-VIO repo now to accompany this PR. Since the tag for the corresponding commit exists, we can test this PR in CI, and then merge the equivalent Kimera-VIO PR somewhat independently.

All versions will be easier to track once we integrate the refactored configurations for #231 and #54. Each external module has its own `.yaml` configuration specifying the tag so we can update versions in a consistent manner across plugins and configurations.

Reply:
Nice!